Unit Tests Breaks with Dashed Path

If there is a dash in the path containing RAD code (e.g. /src/rad-temp/), the following test breaks:

https://github.com/spacetelescope/rad/blob/b9c2b7542209bdb939ca50ce0cf2f33bd6a50092/tests/test_latest.py#L33

`>       assert len(str(latest_path).split("/rad/resources/")[-1].split("-")) == 1`
`E       AssertionError: assert 2 == 1`
`E        +  where 2 = len(['/src/rad', 'temp/rad/latest/SSC/MSOS/keywords/msos_reference_frame.yaml'])`
`E        +    where ['/src/rad', 'temp/rad/latest/SSC/MSOS/keywords/msos_reference_frame.yaml'] = <built-in method split of str object at 0x110d34390>('-')`
`E        +      where <built-in method split of str object at 0x110d34390> = '/src/rad-temp/rad/latest/SSC/MSOS/keywords/msos_reference_frame.yaml'.split`

--- a/tests/test_latest.py
+++ b/tests/test_latest.py
@@ -30,7 +30,7 @@ class TestLastestResources:
         Check that the file name of the schema matches the schema ID WITHOUT the version number suffix.
         """
         # Check that the file name does not contain a version number
-        assert len(str(latest_path).split("/rad/resources/")[-1].split("-")) == 1
+        assert len(str(latest_path).split("/rad/latest/")[-1].split("-")) == 1
 
         # Check that the file name is consistent with the schema ID
         uri = yaml.safe_load(latest_path.read_bytes())["id"]

